docs/io.cozy.files.md

```diff
@@ -213,7 +213,8 @@ They can have `metadata`:
 - `target.mime` is the mime-type of the destination of the link (when it is a
   file)
 - `target.app` is the slug of the destination app (internal links only).
-- `icon` {string?} contains the svg icon.   
+- `icon` {string?} contains the svg icon base64 encoded.
```

Comment:
This is not 100% true, we can have directly the binary here (and then we have to encode it to base64) or we can have the base64.

See https://github.com/cozy/cozy-home/pull/1919/files#r1156901276